x-pack/plugin/src/test/resources/rest-api-spec/api/async_search.submit.json

```diff
@@ -34,7 +34,7 @@
       },
       "keep_on_completion":{
         "type":"boolean",
-        "description":"Control whether the response should be stored in the cluster if it completed within the provided [wait_for_completion] time (default: true)",
+        "description":"Control whether the response should be stored in the cluster if it completed within the provided [wait_for_completion] time (default: false)",
```

Comment:
LOL, I changed the default one line below but not in the description, thanks for fixing